File: test-sources/test-fill-stdout.pas
(* $Id$
 *
 * Floods stdout and should fail with TIME-LIMIT or RUN-ERROR
 * depending on whether timelimit or filesize limit is reached first.
 *
 * @EXPECTED_RESULTS@: TIME-LIMIT,RUN-ERROR
 *)

program fillstdout(input, output);

begin
   While 1=1 Do
   begin
	  writeln('Fill stdout with nonsense, to test filesystem stability.');
   end;
end.

File: example_problems/hello/submissions/time_limit_exceeded/test-fill-stdout.cc
/*
 * Floods stdout and should fail with TIMELIMIT, since output is
 * automatically truncated at the filesize limit.
 *
 * @EXPECTED_RESULTS@: TIMELIMIT
 */

#include <iostream>
#include <string>

int main()
{
	while ( 1 ) std::cout << "Fill stdout with nonsense, to test filesystem stability.\n";

	return 0;
}
